ee/fleet-agent-downloader/package.json

```
@@ -28,9 +28,9 @@
   "scripts": {
     "start": "NODE_ENV=production node app.js",
     "test": "npm run lint && npm run custom-tests && echo 'Done.'",
-    "lint": "./node_modules/eslint/bin/eslint.js . --max-warnings=0 --report-unused-disable-directives && echo '✔  Your .js files look so good.' && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/**/**/**/*.ejs && echo '✔  So do your .ejs files.' && ./node_modules/lesshint/bin/lesshint assets/styles/ --max-warnings=0 && echo '✔  Your .less files look good, too.'",
+    "lint": "./node_modules/eslint/bin/eslint.js . --max-warnings=0 --report-unused-disable-directives && echo '\u2714  Your .js files look so good.' && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/**/**/*.ejs && ./node_modules/htmlhint/bin/htmlhint -c ./.htmlhintrc views/**/**/**/**/**/**/*.ejs && echo '\u2714  So do your .ejs files.' && ./node_modules/lesshint/bin/lesshint assets/styles/ --max-warnings=0 && echo '\u2714  Your .less files look good, too.'",
```

ESLint

[error] Replace `\u2714··Your·.js·files·look·so·good.'·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/**/**/**/*.ejs·&&·echo·'\u2714··So·do·your·.ejs·files.'·&&·./node_modules/lesshint/bin/lesshint·assets/styles/·--max-warnings=0·&&·echo·'\u2714` with `✔··Your·.js·files·look·so·good.'·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/**/**/*.ejs·&&·./node_modules/htmlhint/bin/htmlhint·-c·./.htmlhintrc·views/**/**/**/**/**/**/*.ejs·&&·echo·'✔··So·do·your·.ejs·files.'·&&·./node_modules/lesshint/bin/lesshint·assets/styles/·--max-warnings=0·&&·echo·'✔`

(prettier/prettier)